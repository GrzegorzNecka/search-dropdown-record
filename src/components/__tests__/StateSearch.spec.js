import React from 'react';
import StateSearch from '../StateSearch';
import renderer from 'react-test-renderer';
import { SearchStateContext } from '../../hooks/useSearchState';

describe('StateSearch', () => {
  it('Match to snapshot', () => {
    const tree = renderer.create((
      <SearchStateContext.Provider value={{
        query: "test query",
        onSetQuery: jest.fn(),
        list: [],
        onPick: jest.fn(),
      }}>u
        <StateSearch />
      </SearchStateContext.Provider>
    )).toJSON();

    expect(tree).toMatchSnapshot();
  });
})