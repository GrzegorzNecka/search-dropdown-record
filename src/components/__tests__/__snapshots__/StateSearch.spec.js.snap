// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateSearch Match to snapshot 1`] = `
<div
  className="field"
>
  <div
    className="control"
  >
    <div
      className="dropdown is-active"
    >
      <div
        className="dropdown-trigger"
      >
        <input
          className="input"
          onChange={[Function]}
          onFocus={[Function]}
          onKeyUp={[Function]}
          placeholder="wyszukaj..."
          type="text"
          value="test query"
        />
      </div>
    </div>
  </div>
</div>
`;
